diff -ru elfutils-0.152/configure.ac elfutils-0.152-stamp/configure.ac
--- elfutils-0.152/configure.ac	2011-02-15 15:30:12.000000000 +0100
+++ elfutils-0.152-stamp/configure.ac	2011-08-24 08:00:01.725025320 +0200
@@ -29,7 +29,7 @@
 AM_MAINTAINER_MODE
 
 dnl Unique ID for this build.
-MODVERSION="Build on $(hostname) $(date +%FT%R:%S%z)"
+MODVERSION="Built on OBS, see RPM information for build date and host"
 AC_SUBST([MODVERSION])
 AC_DEFINE_UNQUOTED(MODVERSION, "$MODVERSION")
 AH_TEMPLATE([MODVERSION], [Identifier for modules in the build.])
diff -ru elfutils-0.152/src/nm.c elfutils-0.152-stamp/src/nm.c
--- elfutils-0.152/src/nm.c	2010-06-29 06:09:31.000000000 +0200
+++ elfutils-0.152-stamp/src/nm.c	2011-08-24 08:01:02.052577571 +0200
@@ -134,7 +134,7 @@
 
 #define INTERNAL_ERROR(fname) \
   error (EXIT_FAILURE, 0, gettext ("%s: INTERNAL ERROR %d (%s-%s): %s"),      \
-	 fname, __LINE__, PACKAGE_VERSION, __DATE__, elf_errmsg (-1))
+	 fname, __LINE__, PACKAGE_VERSION, "built on OBS, check rpm -q --full elfutils for information", elf_errmsg (-1))
 
 
 /* Internal representation of symbols.  */
diff -ru elfutils-0.152/src/objdump.c elfutils-0.152-stamp/src/objdump.c
--- elfutils-0.152/src/objdump.c	2010-06-29 06:09:31.000000000 +0200
+++ elfutils-0.152-stamp/src/objdump.c	2011-08-24 08:01:02.057577533 +0200
@@ -103,7 +103,7 @@
 
 #define INTERNAL_ERROR(fname) \
   error (EXIT_FAILURE, 0, gettext ("%s: INTERNAL ERROR %d (%s-%s): %s"),      \
-	 fname, __LINE__, PACKAGE_VERSION, __DATE__, elf_errmsg (-1))
+	 fname, __LINE__, PACKAGE_VERSION, "built on OBS, check rpm -q --full elfutils for information", elf_errmsg (-1))
 
 
 /* List of sections which should be used.  */
diff -ru elfutils-0.152/src/size.c elfutils-0.152-stamp/src/size.c
--- elfutils-0.152/src/size.c	2010-06-29 06:09:31.000000000 +0200
+++ elfutils-0.152-stamp/src/size.c	2011-08-24 08:01:02.112577125 +0200
@@ -115,7 +115,7 @@
 
 #define INTERNAL_ERROR(fname) \
   error (EXIT_FAILURE, 0, gettext ("%s: INTERNAL ERROR %d (%s-%s): %s"),      \
-	 fname, __LINE__, PACKAGE_VERSION, __DATE__, elf_errmsg (-1))
+	 fname, __LINE__, PACKAGE_VERSION, "built on OBS, check rpm -q --full elfutils for information", elf_errmsg (-1))
 
 
 /* User-selectable options.  */
diff -ru elfutils-0.152/src/strip.c elfutils-0.152-stamp/src/strip.c
--- elfutils-0.152/src/strip.c	2011-01-12 23:34:55.000000000 +0100
+++ elfutils-0.152-stamp/src/strip.c	2011-08-24 08:01:02.128577005 +0200
@@ -122,7 +122,7 @@
 
 #define INTERNAL_ERROR(fname) \
   error (EXIT_FAILURE, 0, gettext ("%s: INTERNAL ERROR %d (%s-%s): %s"),      \
-	 fname, __LINE__, PACKAGE_VERSION, __DATE__, elf_errmsg (-1))
+	 fname, __LINE__, PACKAGE_VERSION, "built on OBS, check rpm -q --full elfutils for information", elf_errmsg (-1))
 
 
 /* Name of the output file.  */
